src/governance/governance.cpp

```diff
@@ -1485,7 +1484,7 @@ void CGovernanceManager::UpdatedBlockTip(const CBlockIndex* pindex, CConnman& co
         return;
     }
 
-    const auto sb_opt = CreateSuperblockCandidate(pindex->nHeight);
+    const auto sb_opt = mn_activeman != nullptr ? CreateSuperblockCandidate(pindex->nHeight) : std::nullopt;
```

Comment:
see comment for `CreateGovernanceTrigger` for better mn_activeman check

Reply:
Resolved in latest push!